### PR TITLE
Hypercare alterations

### PR DESCRIPTION
**Pull request by erickmartins**
A lot of minor changes. Improved xlsx parsing, dealing better with errors, etc etc

### CODE REVIEW

File: auto_import.sh [hunk 1]
```diff
@@ -59,10 +60,10 @@ for dir in $(sudo -u $user find $folder -mindepth 1 -maxdepth 1 -type d -mmin +6
         if [ $modified -gt 1 ]; then
             email=true
         fi
-
+        echo "Will I send an email? $email"
```
Comment (mellertd) [inline auto_import.sh:63]: what is this line for? looks like a debugging line?

File: auto_import.sh [hunk 2]
```diff
@@ -81,13 +82,38 @@ for dir in $(sudo -u $user find $folder -mindepth 1 -maxdepth 1 -type d -mmin +6
             echo "(that does NOT mean that all files in your SPREADSHEET were imported; it is your" && \
             echo -e "responsibility to check all files from your submission were present in the folder.)\n" )
         fi
+        echo "logfile should be at $logfile"
+        if sudo -u $user [ -f "$logfile" ]; then
+            echo "I have found the logfile"
```
Comment (mellertd) [inline auto_import.sh:87]: more debugging? Or does this go into our logs?

File: auto_import.sh (hunk 1 above)
Comment (erickmartins) [reply auto_import.sh:63]: It is, but it's only saved on our internal cronlogs, not sent to the user. Can still delete it though

File: auto_import.sh (hunk 2 above)
Comment (erickmartins) [reply auto_import.sh:87]: Same as above - it goes into our internal logs, not to user logs.

**Review by mellertd — APPROVED** [review]
looks good to me

**Pull request merged**